src/plugins/path/path.c

```diff
@@ -206,8 +207,15 @@ static int validatePermission (Key * key, Key * parentKey)
 	}
 
 	// Change back to initial effective IDs
-	seteuid (currentUID);
-	setegid (currentGID);
+	int euidResult = seteuid (currentUID);
+	int egidResult = setegid (currentGID);
+
+	if (euidResult != 0 || egidResult != 0)
+	{
+		ELEKTRA_SET_ERRORF (202, parentKey, "Could not change back to user before. This case should not happen", name,
```

Comment:
Please go into the users perspective. Users do not know that any user was changed and what they should do now (report a bug at issues.libelektra.org)